tuned/plugins/plugin_cpu.py

```diff
@@ -708,6 +716,8 @@ def _set_pm_qos_resume_latency_us(self, pm_qos_resume_latency_us, device, sim):
 				log.warning("Invalid latency specified: '%s'" % str(pm_qos_resume_latency_us))
 				return None
 		if not sim:
+			if not self._check_pm_qos_resume_latency_us(device):
+				return None
```

Comment:
I think it should return `None` here even if `sim` is `True` otherwise it may break verification.

Reply:
You're probably right, we're not 100% consistent with this behavior across plugins.